pkg/cmd/dcos.go

```diff
@@ -9,16 +9,20 @@ import (
 
 	"github.com/dcos/dcos-cli/pkg/config"
 	"github.com/spf13/cobra"
+	"io"
```

Comment:
this should go in the previous import section (https://github.com/golang/go/wiki/CodeReviewComments#imports).